pkg/mattermost/mattermost.go

```diff
@@ -233,15 +233,15 @@ func GenerateDeployment(mattermost *mattermostv1alpha1.ClusterInstallation, dbIn
 			ImagePullPolicy: corev1.PullIfNotPresent,
 			Command: []string{
 				"/bin/sh", "-c",
-				fmt.Sprintf("mc config host add localminio http://%s $(MINIO_ACCESS_KEY) $(MINIO_SECRET_KEY) && mc mb localminio/%s -q -p", minioURL, mattermost.Name),
+				fmt.Sprintf("mc config host add localminio http://%s $(accesskey) $(secretkey) && mc mb localminio/%s -q -p", minioURL, mattermost.Name),
```

Comment:
ENV names are `accessKey` & `secretKey`

Reply:
I'm restoring this to the way it was. I did several tests because of a secret change between minio versions and changed this, but it was good the way it were.